<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoiceWindow</class>
 <widget class="QMainWindow" name="ChoiceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>524</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="217">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="217">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="63">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>ChoiceWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Fira Mono</family>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RecommUi</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_genre">
          <property name="text">
           <string>What genre are you interested in?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="genre_crime">
            <property name="text">
             <string>Crime</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_ficiton">
            <property name="text">
             <string>Fiction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_fantasy">
            <property name="text">
             <string>Fantasy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_non_fiction">
            <property name="text">
             <string>Non-Fiction</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_mystery">
            <property name="text">
             <string>Mystery</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="genre_classic">
            <property name="text">
             <string>Classic</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_manga">
            <property name="text">
             <string>Manga</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_sci_fi">
            <property name="text">
             <string>Sci-Fi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_historical">
            <property name="text">
             <string>Historical</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_horror">
            <property name="text">
             <string>Horror</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCheckBox" name="genre_thriller">
            <property name="text">
             <string>Thriller</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_humor">
            <property name="text">
             <string>Humor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_suspense">
            <property name="text">
             <string>Suspense</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_romance">
            <property name="text">
             <string>Romance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="genre_cookbook">
            <property name="text">
             <string>Cookbook</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_year">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Year Released</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QSpinBox" name="spinBox_year">
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="217">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="217">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="63">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="font">
             <font>
              <weight>50</weight>
              <bold>false</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>false</bool>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
            </property>
            <property name="minimum">
             <number>-720</number>
            </property>
            <property name="maximum">
             <number>2018</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="slider_year">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>1300</width>
              <height>1300</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>-720</number>
            </property>
            <property name="maximum">
             <number>2018</number>
            </property>
            <property name="sliderPosition">
             <number>-720</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_rating">
          <property name="text">
           <string>Rating</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_stars">
            <property name="text">
             <string>Stars:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_rating">
            <property name="minimumSize">
             <size>
              <width>10</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>20</height>
             </size>
            </property>
            <property name="palette">
             <palette>
              <active>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="217">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </active>
              <inactive>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="217">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </inactive>
              <disabled>
               <colorrole role="Text">
                <brush brushstyle="SolidPattern">
                 <color alpha="63">
                  <red>0</red>
                  <green>0</green>
                  <blue>0</blue>
                 </color>
                </brush>
               </colorrole>
              </disabled>
             </palette>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::PlusMinus</enum>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDial" name="dial_rating">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>4</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="button_find">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string> Find</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_userlist">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string> Read Later</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_refresh">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string> Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_exit">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string> Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecommUi">
    <property name="title">
     <string>&amp;RecommUi</string>
    </property>
   </widget>
   <addaction name="menuRecommUi"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
